amundsen_application/static/js/components/TableDetail/WatermarkLabel/index.tsx

```diff
@@ -43,18 +43,24 @@ class WatermarkLabel extends React.Component<WatermarkLabelProps> {
     }
 
     return (
-      <>
-        <div className="range-labels body-2">
-          {LOW_WATERMARK_LABEL}
-          <br />
-          {HIGH_WATERMARK_LABEL}
-        </div>
-        <div className="range-dates body-2">
-          {low && this.formatWatermarkDate(low)}
-          <br />
-          {high && this.formatWatermarkDate(high)}
-        </div>
-      </>
+      <div className="date-ranges">
+        {low && (
```

Comment:
Would want to verify the change this makes to the design is okay before changing it without it being requested.

Reply:
It looks the same, but I can double check!

Reply:
It's minor to me (yet I don't want to make assumptions about what design cares about vs not) but its how the date texts align stacked on each other. 

**From Figma:**
<img src='https://user-images.githubusercontent.com/1790900/84737337-93d0ad00-af5c-11ea-8a3d-1613db3d28ed.png' width='40%' />

**This PR:**
<img src='https://user-images.githubusercontent.com/1790900/84737301-7e5b8300-af5c-11ea-9524-64a6153da148.png' width='40%' />

Reply:
Oh, good point.  
I could fix it by setting a fixed width on the 'date-range-label', but that won't be flexible if we end up allowing translations.

Do you want me to fix it?

Reply:
Yes let's fix it for now to avoid making a design change just because we wanted to clean stuff up. Then if/when we allow translations we can deal with that issue by asking for an update design that considers this.

Reply:
OK, fixing it.